 # a cropped pointcloud representing some number of items
sensor_msgs/PointCloud2 cloud
# The id of the item if the cloud represents a single item, else the ids of the items from left to right
string[] labels
